CMakeLists.txt

```diff
@@ -33,6 +33,10 @@ if(CMAKE_CROSSCOMPILING)
     cmake_minimum_required(VERSION 3.3) # Version introducing CROSSCOMPILING_EMULATOR
 endif()
 
+if(MSVC)
```

Comment:
Should be `WIN32`, that way building in `Debug` on windows using Ninja will work as expected.